### ISSUE TITLE
Re-produce COCO-O results

### ISSUE THREAD

**Issue by JohannesTheo** [alibaba/easyrobust#21]
Hey there,

I try to re-produce some of the results in the COCO-O paper in order to use the dataset myself. Are the reported COCO mAP scores for val2017 or test-dev2017? I see that you have used mmdet and since I get slightly different results with pre-trained models from torchvision, I was wondering if this is due to different id-test set or due to small differences in the trained models.

Example: For [Faster R-CNN ResNet-50 FPN](https://pytorch.org/vision/main/models/faster_rcnn.html) I get:

| | coco mAP | sketch | weather | cartoon | painting | tattoo | handmake | avg. | ER |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
|paper| 37.4 | 9.8 | 25.1 | 13.9 | 23.3 | 10.1 | 16.3 | 16.4 | -0.41 |
|torchvision| 37.0 (val2017) | 8.4 | 27.3 | 13.4 | 22.2 | 11.5 | 15.9 | 16.44 | -0.21 |

 Thanks and keep up the great work!
Johannes

**Comment by vtddggg** [comment]
We use val2017 set for the COCO mAP scores reported in the paper. 

To reproduce, it will be better to use the mmdet weights of R-50-FPN with 1x LR schduled in [this table](https://github.com/open-mmlab/mmdetection/tree/main/configs/faster_rcnn#results-and-models). We cannot ensure the pre-trained weights from torchvision is totally same with mmdet weights.

**Comment by JohannesTheo** [comment]
Thanks a lot for the response! :) 

I think it's totally normal that different weights for the same model will give slightly different results. I just wanted to make sure that I'm using the benchmark right and do correct comparisons.
